draft-ietf-quic-qlog-main-schema.md

```diff
@@ -659,68 +666,186 @@ Example qlog event:
 
 ## Timestamps {#time-based-fields}
 
-An event's "time" field indicates the timestamp at which the event occurred. Its value is
-typically the Unix timestamp since the 1970 epoch (number of milliseconds since
-midnight UTC, January 1, 1970, ignoring leap seconds). However, qlog supports two
-more succinct timestamps formats to allow reducing file size. The employed format
-is indicated in the "time_format" field, which allows one of three values:
-"absolute", "delta" or "relative".
+Each event MUST include a "time" field to indicate the timestamp that it
+occurred. It is a duration measured from some point in time, in units of ???.
```

Comment:
```suggestion
occurred. It is a duration measured from some point in time; its units depend on the type of clock chosen and system used. Typically however, a millisecond resolution is employed.
```

Reply:
I don't understand this bit. If I have a float64 field, and I decide to use it to log "1.5" to mean one and a half seconds, but the logger tool thinks I meant one and a half milliseconds, then there's an interop issue. So I think we need to pick one form of "units of x with fractional Y", or add a field to articulate the concrete units and precision.